apps/marketing/src/components/(marketing)/CTA.tsx

```diff
@@ -0,0 +1,24 @@
+import Link from 'next/link';
+
+import { Button } from '@documenso/ui/primitives/button';
+
+export default function CTA() {
+  return (
+    <div className="bg-background shadow-foreground/5 border-documenso mt-12 flex flex-col items-center justify-center rounded-lg border-4 border-double p-8 shadow-lg hover:shadow-md">
```

Comment:
also take classname as a prop, we never want to have `mt-12` or similar in the component itself since it's usually dependent on the layout/view/page its in